certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -28,6 +28,7 @@
 from certbot_nginx import parser
 from certbot_nginx import tls_sni_01
 from certbot_nginx import http_01
+from certbot_nginx import obj # pylint: disable=unused-import, no-name-in-module
```

Comment:
nit: We shouldn't need `no-name-in-module` here.

Reply:
Good catch; done!